.build_rtd_docs/Doxyfile

```diff
@@ -37,7 +37,7 @@ PROJECT_NAME           = "MODFLOW 6"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = "version 6.2.2"
+PROJECT_NUMBER         = "version 6.3.0"
```

Comment:
this should probably be fixed?

Reply:
That is updated by one of our scripts when doxygen is built on CI. But should update at some point so now is good.